shared/js/media/remote_controls.js

```diff
@@ -77,8 +78,7 @@ var REMOTE_CONTROLS = {
  * @constructor
  */
 function MediaRemoteControls() {
-  this.bluetooth = navigator.mozBluetooth;
-  this.defaultAdapter = null;
+  this._btHelper = null;
```

Comment:
Please use full name `this._bluetoothHelper` here. Full name would be more readable in the future. Thanks.